src/script/arch-update-tray.py

```diff
@@ -22,6 +23,26 @@
     log.error("Statefile does not exist: %s", STATE_FILE)
     sys.exit(1)
 
+# Find translations
+paths = []
+if 'XDG_DATA_DIRS' in os.environ:
```

Comment:
Correct me if I'm wrong, but there's a flaw in the logic here.

If someone has XDG_DATA_DIRS defined to a custom dir but still used the default `/usr/local` prefix at install (or `/usr` for the AUR package), the translation will be reported "not found".
I think we should use the same logic as for the Desktop file, like "if XDG_DATA_DIRS is defined, look into it. If not found there, than if XDG_DATA_HOME is defined, look into it. If not found there [...]". Right?